hydrolib/core/dflowfm/extold/models.py

```diff
@@ -228,21 +236,55 @@ class ExtOldInitialConditionQuantity(StrEnum):
     Initial Condition quantities:
         initialwaterlevel, initialsalinity, initialsalinitytop, initialtemperature,
         initialverticaltemperatureprofile, initialverticalsalinityprofile, initialvelocityx,
-        initialvelocityy, initialvelocity, initialsalinitytopuse
+        initialvelocityy, initialvelocity
+
+    If there is a missing quantity that is mentioned in the "Accepted quantity names" section of the user manual
+    [Sec.C.5.3](https://content.oss.deltares.nl/delft3dfm1d2d/D-Flow_FM_User_Manual_1D2D.pdf#subsection.C.5.3).
+    and [Sec.D.3](https://content.oss.deltares.nl/delft3dfm1d2d/D-Flow_FM_User_Manual_1D2D.pdf#subsection.D.3).
+    please open and issue in github.
     """
 
     # Initial Condition fields
+    BedLevel = "bedlevel"
+    BedLevel1D = "bedlevel1D"
+    BedLevel2D = "bedlevel2D"
+
     InitialWaterLevel = "initialwaterlevel"
+    InitialWaterLevel1D = "initialwaterlevel1d"
+    InitialWaterLevel2D = "initialwaterlevel2d"
+
     InitialSalinity = "initialsalinity"
     InitialSalinityTop = "initialsalinitytop"
+    InitialSalinityBot = "initialsalinitybot"
+    InitialVerticalSalinityProfile = "initialverticalsalinityprofile"
+
     InitialTemperature = "initialtemperature"
     InitialVerticalTemperatureProfile = "initialverticaltemperatureprofile"
-    InitialVerticalSalinityProfile = "initialverticalsalinityprofile"
 
+    initialUnsaturatedZoneThickness = "initialunsaturatedzonethickness"
     InitialVelocityX = "initialvelocityx"
     InitialVelocityY = "initialvelocityy"
     InitialVelocity = "initialvelocity"
-    InitialSalinityTopUse = "initialsalinitytopuse"
+    InitialWaqBot = "initialwaqbot"
+
+    @classmethod
+    def _missing_(cls, value):
```

Comment:
Nice to have learned about this Pydantic overriding/runtime adding of model fields.